x-pack/plugins/fleet/common/types/models/epm.ts

```diff
@@ -330,7 +330,7 @@ export interface RegistryDataStreamPrivileges {
   indices?: string[];
 }
 
-export type RegistryVarType = 'integer' | 'bool' | 'password' | 'text' | 'yaml' | 'string';
+export type RegistryVarType = 'integer' | 'bool' | 'password' | 'text' | 'yaml';
```

Comment:
I removed this because it looks like it was intended to be removed in https://github.com/elastic/kibana/pull/94174/files but wasn't (I think this was an accident)

cc @jen-huang